packages/experimental/Badge/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@fluentui-react-native/badge",
+  "version": "0.0.1",
+  "description": "A cross-platform Badge component using the Fluent Design System. A badge is an additional visual discriptor for UI elements.",
+  "license": "MIT",
+  "author": "Microsoft <fluentuinativeowners@microsoft.com>",
+  "homepage": "https://github.com/microsoft/fluentui-react-native",
+  "main": "src/index.ts",
+  "module": "src/index.ts",
+  "typings": "lib/index.d.ts",
+  "onPublish": {
+    "main": "lib-commonjs/index.js",
+    "module": "lib/index.js"
+  },
+  "scripts": {
+    "build": "fluentui-scripts build",
+    "just": "fluentui-scripts",
+    "clean": "fluentui-scripts clean",
+    "lint": "fluentui-scripts eslint",
+    "depcheck": "fluentui-scripts depcheck",
+    "test": "fluentui-scripts jest",
+    "update-snapshots": "fluentui-scripts jest -u",
+    "prettier": "fluentui-scripts prettier",
+    "prettier-fix": "fluentui-scripts prettier --fix true"
+  },
+  "dependencies": {
+    "@fluentui-react-native/experimental-text": "^0.7.9",
+    "@fluentui-react-native/framework": "0.7.0",
+    "@fluentui-react-native/icon": "^0.11.3",
+    "@fluentui-react-native/interactive-hooks": "^0.11.11",
+    "@fluentui-react-native/theme-tokens": "^0.11.4",
+    "@fluentui-react-native/tokens": "^0.11.4",
+    "@fluentui-react-native/use-styling": "^0.8.0",
+    "@office-iss/react-native-win32": "^0.66.3"
```

Comment:
I think this should be in peer or devdependencies. Otherwise downstream people who consume this for say, a macOS bundle, might also include rn-win32

Reply:
Speaking of, react-native-macos should also be added to the dev/peer dependency list...

Reply:
See other comment, do we need react-native-win32 specifically?

If we do need it, it should be using a version of 0.64.x to align with other react native versions, otherwise there may be API alignment issues.

Reply:
We don't need it. This package was from PersonaCoin. I removed it